PocketKit/Sources/PocketKit/MyList/ItemsList/SwiftUI/Components/ActionButton.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ActionButton.swift
+//  
+//
+//  Created by Ky B Hamilton on 12/7/22.
+//
+
+import SwiftUI
+import Textile
+
+struct ActionButton: View {
+    
+    var itemAction: ItemAction?
+    var selected: Bool = false
+    var trailingPadding: Bool = true
+    
+    init(_ itemAction: ItemAction?, selected: Bool = false, trailingPadding: Bool = true) {
+        self.itemAction = itemAction
+        self.selected = selected
+        self.trailingPadding = trailingPadding
+    }
+    
+    var body: some View {
+        if let action = itemAction, let handler = action.handler, let image = action.image {
+            Button {
+                handler{}
+            } label: {
+                Image(uiImage: image)
+                    .actionButtonStyle(selected: selected, trailingPadding: trailingPadding)
+                    .accessibilityIdentifier(action.accessibilityIdentifier)
+            }
+        }
+    }
+}
```

Comment:
linter is not gonna be happy that all the new files dont have an extra line at the end of it.

Reply:
That should save us 1-2 bytes per file! Consider it a benefit!